migration/cuju-ft-trans-file.c

```diff
@@ -108,6 +117,13 @@ void cuju_ft_trans_flush_buf_desc(void *opaque)
                 //break;
                 continue;
             } else if (ret <= 0) {
+                if(s->check)
+                {
+                    s->check = false;
+                    //printf("cuju_ft_trans_flush_buf_desc!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n");
```

Comment:
Please fix this debug message

Reply:
fix it to printf("primary has sent checkalive header and %s ret <= 0 line:%d\n", __func__, __LINE__);